### PR TITLE
polish the txt2man manual

### PR DESCRIPTION
**Pull request by guijan**
I've made changes to the txt2man manual.

The most innocuous ones are the minor English corrections and making the description like the README.md file in the source tree.
I've changed the line length to be at most 80 characters always, 80 being the "standard" amount of columns in a terminal.

The options are sorted alphabetically and case insensitively by their short forms. The upper case short options don't all come first.
The synopsis is now a complete summary of all the flags.
Some flags which didn't previously list an argument but took one now have the fact that they take arguments clearly specified. I'm not 100% sure I got all of them though.
The manual now uses exclusively the short options while explaining its flags, to encourage their use.

This isn't all but that's the gist of it.

### CODE REVIEW

**Comment by guijan** [comment]
I'm including 2 screenshots of the new man page on my system:
tmux_1627062150-N
tmux_1627062175-N

**Comment by daltomi** [comment]
I really like how it looks 👍🏼

File: man/scrot.txt [hunk 1]
```diff
@@ -2,128 +2,135 @@ NAME
...
-                     A running Composite Manager is needed.
-  -C, --class NAME   Window class name. Associative with options: -k.
-  -S, --script CMD   Imlib2 script commands.
+  -a, --autoselect X,Y,W,H  Non-interactively choose a rectangle starting at
+                            position X,Y and of W by H resolution.
+  -b, --border              When selecting a window, grab the WM's border too.
+                            Use with -s to raise the focus of the window.
+  -C, --class NAME          NAME is a window class name. Associative with -k.
+  -c, --count               Display a countdown when used with -d.
+  -D, --display             Specify the display to use; see X(7).
```
Comment (daltomi) [inline man/scrot.txt:30]: This option requires an argument, maybe `-D, --display NAME`  and at the beginning   `scrot [...] -D [NAME]...`

File: man/scrot.txt [hunk 2]
```diff
@@ -2,128 +2,135 @@ NAME
...
+  -n, --note OPTS           OPTS is a collection of options which specify notes
+                            to bake into the image. See NOTE FORMAT.
+  -o, --overwrite           By default scrot does not overwrite the output
+                            FILE, use this option to enable it.
+  -p, --pointer             Capture the mouse pointer.
+  -q, --quality NUM         NUM must be between 1 and 100. For lossless output
+                            formats, a higher value represents better but slower
+                            compression. For lossy output formats, a higher
+                            value represents higher quality and larger
+                            filesize. Default: 75.
```
Comment (daltomi) [inline man/scrot.txt:49]: Is `filesize` correct in English?

File: man/scrot.txt [hunk 3]
```diff
@@ -2,128 +2,135 @@ NAME
   scrot - command line screen capture utility
 
 SYNOPSIS
-  scrot [options] [file]
+  scrot [-bcDfhkmopsuvz] [-a X,Y,W,H] [-C CMD] [-d SEC] [-e CMD] [-l STYLE]
```
Comment (daltomi) [inline man/scrot.txt:5]: The `-C` option denotes a class name not a command. Maybe `-C [NAME]`?

File: man/scrot.txt (hunk 1 above)
Comment (guijan) [reply man/scrot.txt]: You're right.

File: man/scrot.txt (hunk 2 above)
Comment (guijan) [reply man/scrot.txt]: It's not, good catch.

File: man/scrot.txt (hunk 3 above)
Comment (guijan) [reply man/scrot.txt]: Yes, that was a bad oversight.

**Comment by guijan** [comment]
The previous force-push fixed the merge conflicts with master, this one removes the changes to scrot.1.

It should be ready for further suggestions or merging now.

File: man/scrot.txt [hunk 4]
```diff
@@ -2,128 +2,135 @@ NAME
...
+    -x  position (optional)
+    -y  position (optional)
+    -c  color(RGBA) (optional)
+    -a  angle (optional)
 
   Example:
 
-    $ scrot --note "-f '/usr/share/fonts/TTF/DroidSans-Bold/40' -x 10 -y 20 -c 255,0,0,255 -t 'Hi'"
+    $ scrot --note "-f '/usr/share/fonts/TTF/DroidSans-Bold/40' -x 10 \\
+    >   -y 20 -c 255,0,0,255 -t 'Hi'"
```
Comment (daltomi) [inline man/scrot.txt:130]: Do you agree to remove the `>` character?
The option `--note` should be indicated in short form `-n` as all the previous examples do.

**Pull request merged**